docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.            pd.DataFrame(mapped_adata.var).rename_axis("var_names").reset_index(),
</pre><p>var should already be a data frame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>